addon/styles/components/text-field.scss

```diff
@@ -0,0 +1,79 @@
+.text-field {
+  input:focus {
+    outline: 0;
+  }
+
+  input.text-field__input {
```

Comment:
Now that we have fonts, we can declare the font-family here as well (or maybe `inherit` will work?). Otherwise it uses browser input font.